contrib/julia-mode.el

```diff
@@ -131,6 +131,23 @@ This function provides equivalent functionality, but makes no efforts to optimis
              "\\\\"))
         (group "'"))))
 
+(defconst julia-hanging-assignment-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("=" "+=" "-=" "*=" "/=" "//=" "\\=" "^=" "÷=" "%=" "|=" "&=" "$=" "<<=" ">>="
+                        ".+=" ".-=" ".*=" "./=" ".//=" ".\\=" ".^=" ".÷=" ".%="))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
+
+(defconst julia-hanging-operator-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("^" ".^" "//" ".//" "*" "/" "%" "&" "\\" ".*" "./" ".%" ".\\"
+                        "<<" ">>" ">>>" ".<<" ".>>" ".>>>"
+                        "+" "-" "|" "$" ".+" ".-"
+                        ":" ".." "|>"
+                        ">" "<" ">=" "<=" "==" "===" "!=" "!==" "<:" "≠" "≤" "≥"
+                        ".>" ".<" ".>=" ".<=" ".==" ".!="
+                        "&&" "||" "?"))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
```

Comment:
Are these missing many operators from the list [here](https://github.com/JuliaLang/julia/blob/7c4c123d6db4448a1628a8e2d0326761cf39472f/src/julia-parser.scm#L6)